robotPositioner/robotPositioner.h

```diff
@@ -0,0 +1,382 @@
+#include <vector>
```

Comment:
It's always good to put a ``#pragma once`` on the first line to prevent the compiler including header files (especially **large** header files more than once)